internal/endtoend/testdata/count_star/sqlite/go/query.sql.go

```diff
@@ -0,0 +1,32 @@
+// Code generated by sqlc. DO NOT EDIT.
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+)
+
+const countStarLower = `-- name: CountStarLower :one
+SELECT count(*) FROM bar
+`
+
+func (q *Queries) CountStarLower(ctx context.Context) (int64, error) {
+	row := q.db.QueryRowContext(ctx, countStarLower)
+	var count int64
+	err := row.Scan(&count)
+	return count, err
+}
+
+const countStarUpper = `-- name: CountStarUpper :one
+r;
+
+SELECT COUNT(*) FROM bar
+`
```

Comment:
Not sure what's happening here, but this query is invalid.

Reply:
It's something to do with the `--` comment parsing. I'll look into it properly tomorrow!

Reply:
This issue is fixed now and the code generates properly.